src/webgl/context-features.js

```diff
@@ -50,6 +51,29 @@ Object.keys(WEBGL_FEATURES).forEach(key => {
 });
 export {FEATURES};
 
+// Enables feature detection in IE11 due to a bug where gl.getExtension may return true
+// but fail to compile when the extension is enabled in the shader. Specifically,
+// the OES_standard_derivatives extension fails to compile in IE11 even though its included
+// in the list of supported extensions.
+export function canCompileGLGSExtension(gl, cap) {
```

Comment:
This only needs to be compiled once right, not every time we ask? You could save the result in a map and just reuse it if we already have it.

const glslExtension = {};
if (cap in glslExtension) {
  return glslExtension[cap];
} 
...
glslExtension[cap] = canCompile;
}